common/tasks/rosa/hosted-cp/rosa-hcp-deprovision/rosa-hcp-deprovision.yaml

```diff
@@ -42,6 +42,16 @@ spec:
       type: string
       description: Contains status of a pipeline(Succeeded, Failed, Completed, None)
       default: None
+  volumes:
+    - name: aws-secrets-volume
+      secret:
+        secretName: "$(params.aws-secrets)"
+    - name: oras-credentials-volume
```

Comment:
should we also move the oras-credentials and github to https://console-openshift-console.apps.stone-prd-m01.84db.p1.openshiftapps.com/k8s/ns/rhtap-qe-shared-tenant/secrets as well?

Right now we're split between two secret locations

Reply:
the CI I am debugging is from namespace `rhtap-shared-tenant`, you need to check  https://console-openshift-console.apps.stone-prd-m01.84db.p1.openshiftapps.com/k8s/ns/rhtap-shared-tenant/secrets?name=oras-credentials